tests/_helpers/github-files.js

```diff
@@ -0,0 +1,53 @@
+const fs = require( 'fs' );
+const GitHubClient = require ( './github-client' );
+
+async function sendFiles( branch, files ) {
+	const nextFile = files.shift();
+
+	if( nextFile ) {
+		await sendFile( branch, nextFile.src, nextFile.dest );
+		return sendFiles( branch, files );
+	}
+}
+
+async function sendFile( branch, sourceFilePath, destinationFilePath ) {
+	const newContent = fs.readFileSync( sourceFilePath, {
+		encoding: 'base64'
+	} );
+
+	const file = await getFile( destinationFilePath, branch );
+	await commitFile( file.data.sha, newContent, destinationFilePath, branch );
```

Comment:
Maybe `sendFile` could also return a result of `commitFile`? Or anything which will tell that commiting filie was successful or failed?